lib/grape-swagger/doc_methods/move_params.rb

```diff
@@ -18,30 +18,18 @@ def to_definition(path, params, route, definitions)
 
           params_to_move = movable_params(params)
 
-          return (params + correct_array_param(params_to_move)) if should_correct_array?(params_to_move)
-
           params << parent_definition_of_params(params_to_move, path, route)
 
           params
         end
 
         private
 
-        def should_correct_array?(param)
-          param.length == 1 && param.first[:in] == 'body' && param.first[:type] == 'array'
-        end
-
-        def correct_array_param(param)
-          param.first[:schema] = { type: param.first.delete(:type), items: param.first.delete(:items) }
-
-          param
-        end
-
         def parent_definition_of_params(params, path, route)
           definition_name = OperationId.build(route, path)
-          build_definition(definition_name, params)
+          # NOTE: Parent definition is always object
+          @definitions[definition_name] = object_type
```

Comment:
Since it is [impossible](https://github.com/ruby-grape/grape/issues/1730) (in terms of effort/benefit) for the grape to have an array as a root object, I think this is an "okay"-ish change.